src/cls/user/cls_user.cc

```diff
@@ -262,16 +268,20 @@ static int cls_user_remove_bucket(cls_method_context_t hctx, bufferlist *in, buf
     return ret;
   }
 
-  if (entry.user_stats_sync) {
-    dec_header_stats(&header.stats, entry);
-  }
+
+  /* if (entry.user_stats_sync) { */
```

Comment:
i wasn't able to figure out what this `user_stats_sync` stuff was for; are you sure we can remove this check?